pkgs/development/python-modules/confusable-homoglyphs/default.nix

```diff
@@ -0,0 +1,57 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  pythonOlder,
+  setuptools,
+  unittestCheckHook,
+  wheel,
+  click,
+}:
+
+let
+  kebabToSnake = str: builtins.replaceStrings [ "-" ] [ "_" ] str;
+in
+buildPythonPackage rec {
+  pname = "confusable-homoglyphs";
+  version = "3.3.1";
+
+  src = fetchPypi {
+    inherit version;
+    pname = kebabToSnake pname;
+    hash = "sha256-uZUAHJsuG0zqDPXzhAp8eRiKjLutBT1pNXK9jBwexGA=";
+  };
+
+  build-system = [
+    setuptools
+    wheel
+  ];
+
+  disabled = pythonOlder "3.7";
+
+  optional-dependencies = {
+    cli = [ click ];
+  };
+
+  checkInputs = [ unittestCheckHook ] ++ optional-dependencies.cli;
+
+  pythonImportsCheck = [ (kebabToSnake pname) ];
+
+  unittestFlagsArray = [
+    "-s"
+    "tests"
+    "-v"
+  ];
```

Comment:
How about use `pytestCheckHook` instead of `unittestCheckHook`?

Reply:
This module seems to only use `unittest`, since it's an old codebase. Therefore, wouldn't `unittestCheckHook` be appropriate here?

https://git.sr.ht/~valhalla/confusable_homoglyphs/tree/master/item/tests/test_categories.py